pkg/operators/oci/oci.go

```diff
@@ -0,0 +1,218 @@
+// Copyright 2024 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ocioperator
```

Comment:
All operators are in pkg/operators. Should we use a different sub-directory to distinguish layer operators and data operators?

Also, documentation would be useful:
```
// Package ocioperator ...
```